api/src/main/resources/workbench-api.yaml

```diff
@@ -4267,7 +4267,7 @@ paths:
           schema:
             "$ref": "#/definitions/ListDisksResponse"
         404:
-          description: Workspace found
+          description: Workspace not found
```

Comment:
Language issue from previous PR.

Reply:
Not a change suggestion, but my instinct would be to leave these descriptions out unless "Not Found" (the meaning of 404) doesn't carry enough information. I imagine it does in most cases.

Reply:
I'm wondering if 404 is appropriate here. @yonghaoy mentioned this in a previous PR https://github.com/all-of-us/workbench/pull/7678#discussion_r1194255484 The 404 is thrown when the workspace of the disk that you are trying to delete does not exist. It is not saying that the disk that you are trying to delete does not exist.

I would love to get your thoughts on a better approach here.

I am considering something like this in order to throw a more understandable error.

```
  public void deleteDisk(String workspaceNamespace, String diskName) {
    String  googleProject;
    try {
      googleProject =
          workspaceService.lookupWorkspaceByNamespace(workspaceNamespace).getGoogleProject();
    } catch(NotFoundException exception){
      throw new ServerErrorException("Workspace does not exist");
    }
    leonardoNotebooksClient.deletePersistentDisk(googleProject, diskName);
  }
```